CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 
 # SnakeCharm Plugin Changelog
 
+
+## [2024.1.2]
+Released on 22 April 2024
```

Comment:
Please leave release date empty here, the `2024.1.2` isn't released yet and will not be released today

Reply:
I've fixed it